### DISCUSSION TITLE
User Roles and Permissions

### DISCUSSION THREAD

**Discussion by sturgeonc** [doccano/doccano#1152] [Q&A]
We need to configure a proper user and admin structure.  using roles and permission in doccano.
Admins --> Configure and manage platform
Project -->  User and project level management
User -->  annotations etc..

Is this possible and is there documentation on how to configure  these roles?

**Comment by Hironsan** [comment]
The following issue may be relevant to the question:
- https://github.com/doccano/doccano/issues/935

As you have pointed out, currently the word `admin` is confusing(Application admin or project admin). So I think it's necessary to make roles and permissions more clear.
